site/buttons.js

```diff
@@ -1,15 +1,19 @@
-$('.simon-button.green').on('click', function() {
+const greenButton = document.querySelector('.simon-button.green');
```

Comment:
Oh, oops, I thought the linter would catch this.

I've been only using `let` and never using `var` or `const`. I don't dissuade students when they use those, but, e.g., if a student tried to reassign `greenButton` for any reason, good or bad, it's likely they'd spin their wheels for a good amount of time. Better to just take that possibility off the table and wait for a student to ask something like "What's the difference between `let` and `const`? I see `const` in lots of examples."